http/http-advanced-reactive/pom.xml

```diff
@@ -89,5 +98,13 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-undertow</artifactId>
```

Comment:
This is bit dangerous because we lose coverage of RESTEasy Reactive without Undertow and their integration is cumbersome. Can you please move serverlet part to the `servlet-undertow` module? If yo need RR, add it via force deps there.

Reply:
Sure, I can move it to the servlet-undertow module.